src/spec/Paths.php

```diff
@@ -43,7 +43,7 @@ class Paths implements SpecObjectInterface, DocumentContextInterface, ArrayAcces
 
     /**
      * Create an object from spec data.
-     * @param PathItem[]|array[] $data spec data read from YAML or JSON
+     * @param (PathItem|array|null)[] $data spec data read from YAML or JSON
```

Comment:
same here, could you explain this change please?

Reply:
Same reason as above..